runtime/v1/linux/runtime.go

```diff
@@ -395,7 +380,19 @@ func (r *Runtime) loadTasks(ctx context.Context, ns string) ([]*Task, error) {
 	return o, nil
 }
 
-func (r *Runtime) cleanupAfterDeadShim(ctx context.Context, bundle *bundle, ns, id string, pid int) error {
+func (r *Runtime) cleanupAfterDeadShim(ctx context.Context, bundle *bundle, ns, id string) error {
+	t, err := r.tasks.Get(ctx, id)
```

Comment:
We shouldn't do this in `loadTasks`, because task is not added to the task list yet, this will always return.

That's why this is kept out side of `cleanupAfterDeadShim` before.

Reply:
Yes, I also aware this, and restore the code.